File: admin-service/models/item.go
package models

type Item struct {
	ItemId        int64  `gorm:"primary_key;auto_increment" json:"id"`
	SlotId        int16  `json:"slotId"`
	CharacterName string `json:"characterName"`
	ItemTypeId    int16  `json:"itemTypeId"`
	UpgradeLevel  int16  `json:"upgradeLevel"`
	Cost          int32  `json:"cost"`
	NameRu        string `json:"nameRu"`
	DscRu         string `json:"dscRu"`
}

func (Item) TableName() string {
	return "items"
}

File: storage-module/src/main/java/marowak/dev/entity/Item.java
package marowak.dev.entity;

import io.micronaut.core.annotation.Nullable;
import lombok.Builder;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

@Builder
@Entity
@Table(name = "items")
public record Item(
        @Id
        @Column(name = "item_id")
        Long id,

        Integer slotId,

        Integer storageId,

        @Nullable
        @Column(name = "character_name")
        String characterName,

        @Column(name = "item_type_id")
        int itemTypeId,

        Integer upgradeLevel,

        Integer cost,

        String nameRu,

        @Nullable
        String dscRu,

        @Nullable
        Double x,

        @Nullable
        Double y
) {
}
